### PR TITLE
Fix twilio webrtc

### PR DESCRIPTION
**Pull request by lisbakke**
• Don't distribute `Example` dir
• Fix `PropTypes` errors

`Example` dir can't be distributed because it causes this issue https://github.com/facebook/react-native/issues/14209

`PropTypes` was causing errors because it's now a standalone module and not part of `react`

### CODE REVIEW

**Comment by bilby91** [comment]
@lisbakke Thanks for the contribution! Can you take a look at my comments ?

File: react-native-twilio-video-webrtc.podspec [hunk 1]
```diff
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   s.source_files   = 'ios/*.{h,m}'
 
   s.dependency 'React'
-  s.dependency 'TwilioVideo', '>= 1.0.1'
+  s.dependency 'TwilioVideo', '>= 1.3.0'
```
Comment (bilby91) [inline react-native-twilio-video-webrtc.podspec]: I don't think we need to change the constraint. If you want to run `1.3.0` you can add the constraint in your application's Podfile

File: src/TwilioVideo.android.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ import {
   NativeModules,
   findNodeHandle,
 } from 'react-native';
-import React, { PropTypes, Component } from 'react';
-
+import React, { Component } from 'react';
```
Comment (bilby91) [inline src/TwilioVideo.android.js]: What version of react removed the `PropTypes` ? We need to add a constraint to make sure it's compatible

File: package.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "react-native-twilio-video-webrtc",
+  "name": "lisbakke-react-native-twilio-video-webrtc",
   "repository": {
     "type": "git",
     "url": "https://github.com/blackuy/react-native-twilio-video-webrtc.git"
   },
   "homepage": "https://github.com/blackuy/react-native-twilio-video-webrtc",
-  "version": "1.0.2",
+  "version": "1.0.4",
```
Comment (bilby91) [inline package.json]: Why not `1.0.3` ?

**Comment by lisbakke** [comment]
@bilby91 Updated!

**Comment by bilby91** [comment]
@lisbakke Hey, I merged #32 that fixed the prop types issue, you were right with prop-types module filling the previous versions!

I will make update the npm package this week and bump the version.

Thank!

**Pull request closed without merging**